composer.json

```diff
@@ -3,7 +3,7 @@
   "description": "AfterPay module",
   "license": "MIT",
   "require": {
-    "php": ">=7.3",
+    "php": ">=7.1",
```

Comment:
Why reverting? At least 7.2 is safe at this point.

Reply:
I really confused.
What I can do with this problem?

**SprykerCI** build failed if I define **"php": ">=7.1"**. Error: **PHP min version `>=7.1` is not expected `>=7.3`**
When I changed version to `"php": ">=7.3"`, I have error in **ScrutinizerCI**: 
`Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - This package requires php >=7.3 but your PHP version (7.2.13) does not satisfy that requirement.`

Reply:
Use 7.2 then for now.